ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/components/stack/StackComponentView.kt

```diff
@@ -451,10 +453,19 @@ private fun MainStackComponent(
 
     // Show the right container composable depending on the dimension.
     val stack: @Composable (Modifier) -> Unit = { rootModifier ->
+        val scrollState = if (stackState.scrollOrientation == null) {
+            null
+        } else {
+            rememberScrollState()
+        }
```

Comment:
Nit, but we could write this like so: 
```suggestion
        val scrollState = stackState.scrollOrientation?.let { rememberScrollState() }
```

Imo that's more readable, but I realize that's very subjective so not a blocker 😇